src/Sarif.Viewer.VisualStudio/SdkUiUtilities.cs

```diff
@@ -30,6 +33,9 @@ public static class SdkUiUtilities
         private static readonly Guid s_appIdUsesIsolatedCLR = new System.Guid("{074a44d3-1d9d-406c-9f91-c2a4982b1974}");
         internal static readonly Guid s_enviornmentThemeCategory = new System.Guid("624ed9c3-bdfd-41fa-96c3-7c824ea32e3d");
 
+        // Embedded link format: [link text](n) where n is a positive integer
```

Comment:
Spec says `n` is non-negative, not positive (that is, it can be 0).

Reply:
Fixed